app/javascript/components/admin/manage_users/forms/InviteUserForm.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Button, Stack } from 'react-bootstrap';
+import { useForm } from 'react-hook-form';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import FormControl from '../../../shared_components/forms/FormControl';
+import Form from '../../../shared_components/forms/Form';
+import Spinner from '../../../shared_components/utilities/Spinner';
+import useCreateInvitation from '../../../../hooks/mutations/admin/manage_users/useCreateInvitation';
+import { createInvitaionFormFields } from '../../../../helpers/forms/CreateInvitationHelpers';
+
+export default function InviteUserForm({ handleClose }) {
+  const { t } = useTranslation();
+  const methods = useForm();
```

Comment:
It's recommended to provide defaultValues when using `react-hook-form`,
Provided in the first rule in https://react-hook-form.com/api/useform